superset-frontend/src/dashboard/components/nativeFilters/types.ts

```diff
@@ -65,10 +65,16 @@ export interface Filter {
   type: NativeFilterType;
   description: string;
 }
+export interface Divider {
+  id: string;
+  title: string;
+  description: string;
+  type: NativeFilterType;
```

Comment:
I think if you set `type: typeof NativeFilterType.DIVIDER` (or something like that, maybe even just `type: 'DIVIDER'`) then you won't need to do type coercion above anymore